.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
If we need the quotes around "3.10" it would be nice to use quotes for the other versions as well for consistency.